.github/workflows/clippy_pr.yml

```diff
@@ -1,17 +1,6 @@
 name: Clippy Test
 
 on:
-  push:
```

Comment:
I see. That's a bit problematic with the merge queue, because it creates separate branches for each merge queue test. But we don't want to run CI twice on these. I'm not sure if there's a single prefix that we could use here to ignore these.

We could potentially just use a `push:` hook and then somehow skip the workflow if the repository is rust-lang/rust-clippy :thinking:

Reply:
> We could potentially just use a `push:` hook and then somehow skip the workflow if the repository is rust-lang/rust-clippy

That sounds a bit too hacky. TBH I never looked at CI results in my fork. Not sure anymore who requested that. We could do the scream test and just remove it and see if anyone complains :smiling_imp:

Reply:
I agree. Usually the only interaction of CI in forks that I see is people being annoyed by an e-mail flurry spam about failing CI workflows after forking something.

Reply:
I have a PR on my fork where I do all kinds of experiments so I can see the lintcheck results (the lintcheck workflow only runs on PRs), and I believe `@xFrednet` mentioned on Zulip they do the same IIRC. But seeing how the lintcheck.yml file is unchanged, I assume this is unaffected and will still work?

Reply:
Indeed, lintcheck is untouched by this PR.

Reply:
Also: If you open a PR in a fork (instead of just pushing a branch) all checks will run anyway. This is more about not having to open a PR in your fork to get the clippy workflow result.